jbpm-benchmarks/kieserver-performance-tests-jmh/src/main/java/org/jbpm/test/performance/kieserver/KieServerClient.java

```diff
@@ -0,0 +1,101 @@
+package org.jbpm.test.performance.kieserver;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Properties;
+import java.util.Set;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+
+import org.kie.server.api.model.KieContainerResource;
+import org.kie.server.api.model.KieServiceResponse;
+import org.kie.server.api.model.ReleaseId;
+import org.kie.server.api.model.ServiceResponse;
+import org.kie.server.client.KieServicesClient;
+import org.kie.server.client.KieServicesConfiguration;
+import org.kie.server.client.KieServicesFactory;
+import org.kie.server.client.ProcessServicesClient;
+import org.kie.server.client.QueryServicesClient;
+import org.kie.server.client.UserTaskServicesClient;
+
+public class KieServerClient {
+
+	private static final long DEFAULT_TIMEOUT = 30_000;
+
+	public static String containerId = "kieserver-assets";
```

Comment:
Weird formatting.

Reply:
Still the formatting is a little bit weird. I didn't mean to delete lines, but the indentation is weird across the whole file. At least here on GitHub.